.github/workflows/ci.yml

```diff
@@ -0,0 +1,132 @@
+name: CI
+
+on:
+  push:
+    branches: [ master, release ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron:  '0 0 * * *'  # Daily midnight build
+
+
+jobs:
+  lint:
+    name: Linter
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
```

Comment:
not sure if 5 min is enough for all our current e2e tests. I think, now it's at least 15